src/Resources/views/storefront/component/ecs-spb-checkout/ecs-spb-data.html.twig

```diff
@@ -29,15 +29,15 @@
 
         {% block swag_paypal_ecs_spb_data_smart_payment_button %}
             <div data-swag-paypal-smart-payment-buttons="true"
-                 data-swag-pay-pal-smart-payment-buttons-options="{{ spbCheckoutButtonData.jsonSerialize|merge({ 'foo': 'bar' })|json_encode }}">
+                 data-swag-pay-pal-smart-payment-buttons-options="{{ spbCheckoutButtonData.jsonSerialize|merge({ 'isCheckout': true })|json_encode }}">
```

Comment:
Pass `isCheckout` as value in `checkout/confirm/index.html.twig` and `account/order/index.html.twig` instead